cactusbot/commands/magic/command.py

```diff
@@ -1,6 +1,32 @@
 """Manage commands."""
 
 from . import Command
+from ..command import ROLES
+
+_ROLE_NAMES = [ROLES[role].lower() for role in ROLES]
+_REVERSED_ROLES = {v: k for k, v in ROLES.items()}
+
+
+def _role_id(role_name):
+    return _REVERSED_ROLES[role_name] if role_name in _REVERSED_ROLES else -1
```

Comment:
I would use `None` rather than `-1` here, in case `-1` becomes an actual thing at some point™. (Global banned, or muted, or whatever.)